server/src/controllers/index.ts

```diff
@@ -12,5 +14,7 @@ router.post("/user", userValidator.postUser, userController.postUser);
 router.put("/user/:id", userValidator.putUser, userController.putUser);
```

Comment:
Let's comment these out for now. I don't see using them soon. 14-15

Reply:
Yeah all the user stuff is irrelevant for now